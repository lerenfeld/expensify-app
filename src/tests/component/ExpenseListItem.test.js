import React from 'react'
import moment from 'moment'
import {shallow} from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('should render ExpenseListItem with expense',()=>{
  const wrapper = shallow(<ExpenseListItem expenses= {...expenses[0]} />)
  expect(wrapper).toMatchSnapshot();
})

