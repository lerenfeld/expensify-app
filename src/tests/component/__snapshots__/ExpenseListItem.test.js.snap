// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense`] = `
<div>
  <p>
    No Expenses
  </p>
</div>
`;

